packages/integration-tests/tests/connectors.test.ts

```diff
@@ -0,0 +1,26 @@
+import { ConnectorDto } from '@logto/schemas';
+
+import { getConnectorDetails, listConnectors } from '@/connector-api';
+
+describe('list connectors and get details before setting up', () => {
+  it('should succeed to get all connectors and they are all disabled', async () => {
+    const response = await listConnectors();
+    expect(response).toHaveProperty('statusCode', 200);
+    expect(() => {
+      for (const connectorDto of JSON.parse(response.body) as ConnectorDto[]) {
+        expect(connectorDto.enabled).toBeFalsy();
+        expect(connectorDto.config).toEqual({});
+      }
+    }).not.toThrow();
```

Comment:
This logic can be improved once we set the response type like the comments above.